introduction/de/01-introduction.asciidoc

```diff
@@ -0,0 +1,12 @@
+== Einführung
+
+Diese Weiterbildungsserie bietet eine Einführung in InnerSource. 
+InnerSource ist die Anwendung von Open-Source Praktiken und -Prinzipien auf die Softwareentwicklung im Unternehmen. 
+Die InnerSource-Software bleibt dabei Eigentum des Unternehmens, kann jedoch von jedem Mitarbeiter des Unternehmens genutzt, verbessert und erweitert werden. 
+Diese Strategie ermöglicht eine breite und effektive Zusammenarbeit und produziert Software, die sich schnell auf die sich ändernden Anforderungen der verschiedenen internen 
+Stakeholder einstellen und anpassen lässt.
+
+Diese Serie zeigt auf, wie Sie Situationen erkennen in denen die Anwendung von InnerSource Praktiken vorteilhaft sind. 
+Wir werden beispielhaft skizzieren wie InnerSource in diesen Situationen helfen kann und 
```

Comment:
In academic writing, I use `we`/`wir` consistently. In less formal venues (magazines, blogs, maybe the LP articles), I feel unsure about it. Do other's have opinions?